tests/test_blocking_http_server.py

```diff
@@ -0,0 +1,122 @@
+from contextlib import contextmanager
+from copy import deepcopy
+from multiprocessing import Pool
```

Comment:
could you use threadpool? I think starting a subprocess is overkill here.
`from multiprocessing.pool import ThreadPool as Pool`

Reply:
Yes. Done.